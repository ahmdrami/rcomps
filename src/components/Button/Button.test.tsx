import React from 'react'
import { screen, render } from '@testing-library/react'
import StyledButton from './Button'

describe('Button', () => {
  test('should render', () => {
    render(<StyledButton variant="primary">Hi</StyledButton>)
    expect(screen.getByText(/hi/i)).toBeInTheDocument()
  })
})
