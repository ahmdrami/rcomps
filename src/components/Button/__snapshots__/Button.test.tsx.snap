// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should create a snaphot 1`] = `
<DocumentFragment>
  <button
    class="css-vhv7r1"
  >
     Hi
  </button>
</DocumentFragment>
`;
